appveyor.yml

```diff
@@ -11,41 +11,71 @@ environment:
         # See: http://stackoverflow.com/a/13751649/163740
         CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
         GDAL_HOME: "C:\\gdal"
-        PYTHON_ARCH: "64"
+        PYTHONWARNINGS: "ignore:DEPRECATION::pip._internal.cli.base_command"
```

Comment:
This was needed to suppress the pip Python 2.7 deprecation warnings. Currently, it is not needed anymore. I think it does no harm for now, but maybe it is better to remove it now before its purpose gets forgotten in a few years.